src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1156,20 +1156,18 @@ private BaseCrefParameterListSyntax ParseBaseCrefParameterList(bool useSquareBra
         /// <summary>
         /// True if the current token could be the beginning of a cref parameter.
         /// </summary>
-        private bool IsPossibleCrefParameter
+        private bool IsPossibleCrefParameter()
```

Comment:
Why did this change to a method?

Reply:
Aligns with our code style in the rest of the parser. No reason to keep it as a property.

---
In reply to: [159807126](https://github.com/dotnet/roslyn/pull/24023#discussion_r159807126) [](ancestors = 159807126)